### PR TITLE
Various minor things

### PR DESCRIPTION
**Pull request by aledsage**
Mostly improvements to tests (including failing tests for BROOKLYN-328 and BROOKLYN-329). See individual commits for explanation and details.

### CODE REVIEW

File: core/src/test/java/org/apache/brooklyn/util/core/internal/ssh/ExecCmdAsserts.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        fail(expectedCmdRegex + " not matched by any commands in " + actual+(errMsg != null ? "; "+errMsg : ""));
+    }
+
+    public static void assertExecsNotContains(List<? extends ExecCmd> actuals, List<String> expectedNotCmdRegexs) {
+        for (ExecCmd actual : actuals) {
+            assertExecContains(actual, expectedNotCmdRegexs);
+        }
+    }
+    
+    public static void assertExecContains(ExecCmd actual, List<String> expectedNotCmdRegexs) {
```
Comment (neykov) [inline core/src/test/java/org/apache/brooklyn/util/core/internal/ssh/ExecCmdAsserts.java]: Missing `Not`

File: camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/SshCommandSensorYamlTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    @BeforeMethod(alwaysRun=true)
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        RecordingSshTool.clear();
+    }
+    
+    @Test
+    public void testSshCommandSensorWithEffectorInEnv() throws Exception {
+       // TODO Should we automatically trim the stdout?
```
Comment (neykov) [inline camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/SshCommandSensorYamlTest.java]: Already end trimming at https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/core/sensor/ssh/SshCommandSensor.java#L130. I think that's good enough.

**Comment by neykov** [comment]
Can be merged after fixing the `assertExecContains` -> `assertExecNotContains` name.

**Comment by aledsage** [comment]
Thanks @neykov - incorporated comments and squashed. Merging now.

**Pull request merged**